test/expect-tests/dune_rpc_e2e/dune_rpc_e2e.ml

```diff
@@ -9,7 +9,7 @@ module Session = Csexp_rpc.Session
 
 (* enable to debug process stdout/stderr *)
 let debug = false
-let () = if debug then Dune_util.Log.init ~file:(Out_channel stderr) ()
+let () = if debug then Dune_util.Log.init ~file:Stderr ()
```

Comment:
Curious that the only case where `Out_channel` is used was this but it does simplify the code as it is impossible to compare equality of channels so that's neat.